steps/50-lifecycle-lab2-dist-solution/.github/workflows/release-workflow.yml

```
@@ -13,6 +13,7 @@ jobs:
 
   release:
     runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
```

```suggestion
    if: github.ref == 'refs/heads/main'
```
Master is deprecated